src/components/Chat/MessageList/Message/index.js

```diff
@@ -33,20 +47,42 @@ const Message = props => {
   ]
 
   return (
-    <div className={messageWrapperClassList.join(' ')}>
-      {currentUserId !== senderId && (
-        <div className={senderIconClassList.join(' ')}>
-          <i aria-hidden="true" title={sender[0]} />
-          <span className="label">{sender[0]}</span>
+    <>
+      {showDate ? (
+        <div className="show-date-content">
+          <div className="date-divider" />
+          <span className="date-divider-text">{dateText}</span>
+          <div className="date-divider" />
         </div>
+      ) : (
```

Comment:
Conditional rendering is handy for this! You can just render dateText with condition:
```
{showDate && (
    <div className="show-date-content">
          <div className="date-divider" />
          <span className="date-divider-text">{dateText}</span>
          <div className="date-divider" />
    </div>
)}
```

Then there is no need for ` : <></>`

Reply:
Cool! :)